test-suite/js/scalajs-test-suite-2.10-fastopt.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Scala.js Test Suite - Jasmine HTML reporter</title>
+  <title>Scala.js Test Suite</title>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/jasmine/1.3.1/jasmine.css"></link>
+  <link rel="stylesheet" type="text/css" href="test-stylesheet.css">
+  <script type="text/javascript">
+    var __ScalaJSEnv = {"javaSystemProperties": { "scalajs.fastopt-stage": "true" }};
+  </script>
+  <script type="text/javascript" src="./target/scala-2.10/scalajs-test-suite-test-jsdeps.js"></script>
+  <script type="text/javascript" src="./target/scala-2.10/test_definitions.js"></script>
```

Comment:
Ah I see.

A separate configuration is probably not the way to go, because it would have to be duplicated for "every" test configuration that people otherwise use (such as `it`, for example).

Let's keep it the way you did for now. Hopefully we'll get better insight when we eventually generalize to #1709.